x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -203,7 +229,11 @@ export class TelemetryEventsSender {
    * @param channel the elastic telemetry channel
    * @param toSend telemetry events
    */
-  public async sendOnDemand(channel: string, toSend: unknown[]) {
+  public async sendOnDemand(
+    channel: string,
+    toSend: unknown[],
+    axiosInstance: AxiosInstance = axios
+  ) {
```

Comment:
NOTE: The change in these parts such as this part is to send down an `axiosInstance` to capture the preview when hitting the preview endpoint. We might be able to change this in the future if the task runners are no longer doing the joins/coorelations but rather everything is within something such as the receiver to do all the loops.